_posts/2018-11-20-bitcoin-abc-0-18-5.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Bitcoin ABC 0.18.5 Released
+subtitle: Now with deep reorg protection!
+multiLangId: 2018-11-20-bitcoin-abc-0-18-5
+lang: en
+---
+
+We are pleased to announce that Bitcoin ABC 0.18.5 has been released. You can download it at:
+[https://download.bitcoinabc.org/0.18.5/](https://download.bitcoinabc.org/0.18.5/).
+
+We are happy to inform you that the Bitcoin Cash network has been very stable since the upgrade on November 15th.
+We continue to work on keeping users’ funds secure and the network and currency fully functional.
+
+You may have read on different media there are a few miners that continue to threaten the BCH network with chain reorganizations.
+Many see this as a threat to the fundamental functionality of the chain and its use as peer to peer cash.
+We understand the uncertainty this causes for users, businesses, and the market in general.
+
+With this new release, you will know that transactions are immutable after 10 confirmations.
+This protection keeps deep reorganizations from negatively impacting you or your business.
+
+The community has come together stronger than before the protocol upgrade,
+and we hope to continue to foster this collaborative environment as we work together
+to continue to make Bitcoin Cash the best money the world has ever seen.
```

Comment:
repeated "to continue", second one can be removed

Reply:
Thanks. Fixed.